### PR TITLE
ELEMENTS-1275: selectivity dropdown close 10.10

### PR DESCRIPTION
**Pull request by richardsd**
Backport of the https://github.com/nuxeo/nuxeo-elements/pull/312

### CODE REVIEW

**Comment by richardsd** [comment]
Pipeline at https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/629/ 🕐

File: widgets/nuxeo-selectivity.html [hunk 1]
```diff
@@ -613,6 +640,38 @@
...
+          const overflowRegex = /(auto|scroll)/;
+
+          if (style.position !== 'fixed') {
+            let parent = this;
+            while (parent) {
+              if (parent.parentElement) {
+                parent = parent.parentElement;
+              } else if (parent.getRootNode()) {
+                parent = parent.getRootNode();
+                if (parent) {
```
Comment (mnixo) [inline widgets/nuxeo-selectivity.html:656]: [Same thing as in the main PR.](https://github.com/nuxeo/nuxeo-elements/pull/312#discussion_r525234544)

**Review by mnixo — DISMISSED** [review]

**Review by mnixo — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]
LGTM. Please run it through the pipeline if you haven't yet after the force push.

**Comment by richardsd** [comment]
Pipeline 🔵 https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/640/

**Pull request merged**